aip/auth/4115.md

```diff
@@ -5,124 +5,115 @@ state: approved
 created: 2020-08-13
 ---
 
-# Default Credentials From Google Virtual Machines
+# Default Credentials For Google Cloud Virtual Environments
 
-If the client runs on Google virtual machine instance such as [Google App
-Engine][0] or [Compute Engine][1], the auth library **may** obtain the identity
-token that is associated with the instance, and use the identity token as the
-credential to access Google APIs. By using the instance identity, the users no
-longer need to config their credentials explicitly.
+If the client runs on Google cloud virtual environments such as [Google Compute Engine (GCE)][0], 
+[Serverless][1] or [Google Kubernetes Engine (GKE)][2], the auth library **may** leverage 
```

Comment:
micro-nit: Please add Oxford comma after [1] for consistency with the rest of the AIP.

Reply:
Done.